packages/blockchain-wallet-v4-frontend/src/components/Flyout/index.tsx

```diff
@@ -96,6 +96,38 @@ export const FlyoutChild = styled.div`
   height: 100%;
   width: 100%;
 `
+export const Row = styled.div<{
+  color?: string
+}>`
+  padding: 16px 40px;
+  box-sizing: border-box;
+  border-top: 1px solid
+    ${props => (props.color ? props.theme[props.color] : props.theme.grey800)};
+  &:last-child {
+    border-bottom: 1px solid
+      ${props => (props.color ? props.theme[props.color] : props.theme.grey800)};
+  }
+`
+export const Title = styled(Text)<{
+  color: string
+  size?: string
+  weight?: number
+}>`
+  font-size: ${props => props.size};
+  font-weight: ${props => props.weight};
+  color: ${props => props.theme[props.color]};
```

Comment:
@TheLeoB make size and weight required if not providing a default. It looks like a lot of Titles have

size='14px' color='grey600'

So make these the default please and remove from instances of Title that adheres to defaults above already